src/mnemonic/cmd_handler.rs

```diff
@@ -225,6 +230,7 @@ impl KvManager {
         let encoded_count = serialize(&(count + 1))
             .map_err(|_| KvErr(KvError::PutErr(InnerKvError::SerializationErr)))?;
 
+        // If the new count isn't written, tofnd will still work with the latest mnemonic
```

Comment:
yikes!  latest mnemonic only!  We should post an issue to atomize this sequence of kv transactions.